pipelines/LSSTCam-imSim/ApVerifyWithFakes.yaml

```diff
@@ -11,3 +11,12 @@ imports:
   # Can't use $AP_PIPE_DIR/pipelines/LsstCamImSim/ApPipeWithFakes.yaml here
   # because the changes made by that file and ../ApVerifyWithFakes.yaml are
   # hard to separate.
+
+tasks:
+  rbClassify:
+    class: lsst.meas.transiNet.TransiNetTask
+    config:
+      modelFile: testModel
+      weightsFile: '/home/nima/meas_transiNet/tests/data/checkpoint.pth.zip'
+      connections.coaddName: 'goodSeeing'
```

Comment:
Given the implementation of model packages and the current git-repository-based backend, I see this discussion as resolved. Please feel free to reopen it in case I'm mistaken.